### PR TITLE
[DOCFIX] Add documentation about spark-shell login issues

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

File: docs/en/Running-Spark-on-Alluxio.md [hunk 1]
```diff
@@ -150,3 +150,58 @@ executors. For example, if `host1` contains `blockA` and a job using `blockA` is
...
+
+Specifying the Hadoop configuration `fs.alluxio.impl` may also help in resolving this error. `fs.alluxio.impl` should
+be set to `alluxio.hadoop.FileSystem` and if you are using Alluxio in fault tolerant mode, `fs.alluxio-ft.impl` should
+be set to `alluxio.hadoop.FaultTolerantFileSystem`. There are a few alternatives to set these parameters.
+
+#### Update `hadoopConfiguration` in SparkContext
+
+You can update the Hadoop configuration in the SparkContext by:
+
+```
```
Comment (apc999) [inline docs/en/Running-Spark-on-Alluxio.md]: scala or java? not sure if github markdown supports scala syntax higlight

File: docs/en/Running-Spark-on-Alluxio.md [hunk 2]
```diff
@@ -150,3 +150,58 @@ executors. For example, if `host1` contains `blockA` and a job using `blockA` is
...
 However, if `--num-executors=2` and executors are started on `host1` and `host2`, Spark will be smart
 enough to prioritize placing the job on `host1`.
+
+## `Failed to login` Issues with Spark Shell
+
+To run the `spark-shell` with the Alluxio client, the Alluxio client jar will have to be added to the classpath of the
+Spark driver and Spark executors, as described earlier. However, sometimes Alluxio will fail to determine the security
+user and will result in an error message similar to: `Failed to login: No Alluxio User is found.`
+
+### Configure `spark.sql.hive.metastore.sharedPrefixes` for Spark
```
Comment (apc999) [inline docs/en/Running-Spark-on-Alluxio.md]: not clear to me if three options are all required or picking one option is good.
Can we be more specific here?

File: docs/en/Running-Spark-on-Alluxio.md [hunk 3]
```diff
@@ -150,3 +150,58 @@ executors. For example, if `host1` contains `blockA` and a job using `blockA` is
...
+
+Spark uses an isolated classloader to load java classes for accessing the hive metastore. However, the isolated
```
Comment (apc999) [inline docs/en/Running-Spark-on-Alluxio.md]: is this approach only valid for certain Spark versions?

File: docs/en/Running-Spark-on-Alluxio.md (hunk 2 above)
Comment (gpang) [inline docs/en/Running-Spark-on-Alluxio.md]: I labeled this as the recommended solution.

File: docs/en/Running-Spark-on-Alluxio.md (hunk 1 above)
Comment (gpang) [inline docs/en/Running-Spark-on-Alluxio.md]: I added syntax highlighting.

File: docs/en/Running-Spark-on-Alluxio.md (hunk 3 above)
Comment (gpang) [inline docs/en/Running-Spark-on-Alluxio.md]: Updated for Spark 1.4.0+

**Pull request merged**